.travis.yml

```diff
@@ -1,12 +1,24 @@
+env:
+  global:
+    # TODO: We need to add the reporter id here
+    # SEE: https://docs.codeclimate.com/docs/finding-your-test-coverage-token#section-how-to
+    - CC_TEST_REPORTER_ID=<some-id-here>
```

Comment:
This will have to remain like this until we either move away from travis or resolve some issues regarding admin rights in Code Climate.

Reply:
Code coverage in Ninetails is probably one of the bottom 5 things I want to spend time on making work ;)